components/linear_app/sources/common/webhook.mjs

```diff
@@ -31,7 +31,8 @@ export default {
       if (!this.getActions().includes(body?.action)) {
         return false;
       }
-      if (this.projectId) {
+      // projectId does not exist in the returned body data for Comment
+      if (this.projectId && !this.getResourceTypes().includes(constants.RESOURCE_TYPE.COMMENT)) {
```

Comment:
Now that we have sources for issues and comments, would the function `isRelevant` better be implemented inside each individual source?

Reply:
Yes. Done